src/common/config.cpp

```diff
@@ -30,6 +31,10 @@ std::string getLogFilter() {
     return logFilter;
 }
 
+bool debugDump() {
```

Comment:
Ooor just don't make configs global cause it will be annoying to implementing settings editors later